api/src/main/java/edu/cornell/mannlib/vitro/webapp/triplesource/impl/sdb/SDBConnectionSmokeTests.java

```diff
@@ -92,8 +92,8 @@ public void checkDatabaseConnection() {
 		connectionProps.put("user", username);
 		connectionProps.put("password", password);
 
-		try (Connection conn = DriverManager
-				.getConnection(url, connectionProps)) {
+		try{
+			 Connection conn = DriverManager.getConnection(url, connectionProps);
```

Comment:
Was this done for a reason?

The try-with-resources pattern (the original code) means that the resources acquired from the get are released at the end of the block, without having to write finally { } boiler plate code.

This is likely to just consume a connection that isn't released back to the pool.